src/hearings/containers/hearing-actuals/hearing-actuals-view-edit-parties/hearing-actuals-view-edit-parties.component.html

```diff
@@ -0,0 +1,109 @@
+<div class="govuk-width-container ">
```

Comment:
can above src/hearings/containers/hearing-actuals/hearing-actuals-timing/hearing-actuals-timing.component.scss be deleted if it's useless?